src/common.jl

```diff
@@ -63,69 +63,92 @@ end
 ##############################################################################
 # Matrix Methods #############################################################
 
-# sparse case (ABA and BA)
-
-function my_mul_mt!(
-    y::Vector{Float64},
-    A::SparseMatrixCSC{Float64, Int64},
-    x::Vector{Float64},
-)
-    copyto!(y, zeros(Float64, size(y)))
-    for i in 1:size(A, 2)
-        for j in A.colptr[i]:(A.colptr[i + 1] - 1)
-            y[i] += A.nzval[j] * x[A.rowval[j]]
-        end
-    end
-    return
-end
-
-# dense case (PTDF and ABA)
-
-function my_mul_mt!(
-    y::Vector{Float64},
-    A::Matrix{Float64},
-    x::Vector{Float64},
-)
-    y[:] .= transpose(A) * x
-    return
-end
-
-# virtual case: all lines
-
-function my_mul_mt!(
-    y::Vector{Float64},
+# # sparse case (ABA and BA)
```

Comment:
remove this code if it isn't going to be used.

Reply:
done